package.json

```diff
@@ -12,7 +12,8 @@
     "_format": "prettier . --plugin=prettier-plugin-astro --plugin=prettier-plugin-svelte --plugin=prettier-plugin-organize-imports --plugin=prettier-plugin-tailwindcss",
     "format": "npm run _format -- --write",
     "format:check": "npm run _format -- --check",
-    "preinstall": "npx playwright install"
+    "preinstall": "npx playwright install",
+    "test": "vitest"
```

Comment:
vitest was suggested by https://docs.astro.build/en/guides/testing/#unit-and-integration-tests

I gave jest a try and ran into config issues (something about commonjs modules being incompatible with some typescript config).